docs/UsingNavigators.md

```diff
@@ -105,32 +105,34 @@ A more complete example that demonstrates the pushing and popping of routes coul
 ```javascript
 import React, { Component, PropTypes } from 'react';
 import { Navigator, Text, TouchableHighlight, View } from 'react-native';
-
+r
 export default class SimpleNavigationApp extends Component {
   render() {
     return (
       <Navigator
         initialRoute={{ title: 'My Initial Scene', index: 0 }}
         renderScene={(route, navigator) =>
-          <MyScene
-            title={route.title}
-
-            // Function to call when a new scene should be displayed           
-            onForward={ () => {    
-              const nextIndex = route.index + 1;
-              navigator.push({
-                title: 'Scene ' + nextIndex,
-                index: nextIndex,
-              });
-            }}
-
-            // Function to call to go back to the previous scene
-            onBack={() => {
-              if (route.index > 0) {
-                navigator.pop();
-              }
-            }}
-          />
+          return (
```

Comment:
I looked at the two other examples in the beginning of the guide to keep the code style the same and they wrap JSX components in parenthesis, even for 1-liners so I tried to keep the style identical. @hramos but I can change it if you want?